.vscode/settings.json

```diff
@@ -1,3 +1,3 @@
 {
-  "go.buildTags": "e2e unit integration"
+  "go.buildTags": "e2e unit integration gateway_integration policy_integration"
```

Comment:
I;m not sure you need these extra build tags do you? Looks like you are still using "integration" with tests moved into separate directories.

Reply:
yeah cleaned that up